lib/core/src/Cardano/Pool/Metrics.hs

```diff
@@ -352,12 +355,44 @@ newStakePoolLayer tr db@DBLayer{..} nl metadataDir = StakePoolLayer
                     , apparentPerformance
                     , cost = poolCost
                     , margin = poolMargin
+                    , desirability = 0
                     }
                 , poolOwners
                 )
 
-    sortByPerformance :: [(StakePool, a)] -> [(StakePool, a)]
-    sortByPerformance = sortOn (Down . apparentPerformance . fst)
+        rankPools xs =
+            let
+                totalStake = sum $ map (fromIntegral . getQuantity . stake . fst) xs
+                relativeStake p = fromIntegral (getQuantity $ stake p) / totalStake
+
+                -- Does not matter. It only scales the resulting desirability.
+                memberStake = 0.01
+
+                toM (p, _) = P.Pool
+                        { P.leaderStake = 0
+                        , P.stake = relativeStake p
+                        , P.cost = fromIntegral $ getQuantity $ cost p
+                        , P.margin = fromIntegral (getPercentage $ margin p) / 100
+                        , P.recentAvgPerformance = apparentPerformance p
+                        }
+
+            in  map (\((pool, x), des) -> (pool {desirability = des}, x))
+                . sortOn (Down . snd)
+                . (P.nonMyopicMemberRewards jormungandrEnv toM memberStake)
+                $ xs
+
+    -- | Constant needed for ranking pools
+    jormungandrEnv :: P.Env
+    jormungandrEnv = P.Env
+        { P.leaderStakeInfluence = 0
+        , P.punishmentHarshness = 1 -- TODO: What should this be?
```

Comment:
Need to decide on this.

<img width="756" alt="Skärmavbild 2020-01-13 kl  15 22 04" src="https://user-images.githubusercontent.com/304423/72263123-85324280-3618-11ea-9728-a4e2720d0f9a.png">

Reply:
Don't think it is there at all in the latest version of the spec. And not needed in this PR anyway. Removed.